### PR TITLE
Python 3: Fix failing tests in test_handlers.py.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: tools/wptserve/wptserve/handlers.py [hunk 1]
```diff
@@ -175,13 +175,14 @@ def load_headers(self, request, path):
...
         try:
-            with open(headers_path) as headers_file:
+            with open(headers_path, "rb") as headers_file:
                 data = headers_file.read()
         except IOError:
             return []
         else:
             if use_sub:
                 data = template(request, data, escape_type="none")
+            data = data.decode("ascii")
```
Comment (gsnedders) [inline tools/wptserve/wptserve/handlers.py]: Hmm. I suspect we actually want ISO-8859-1 here? We should definitely be able to have non-ASCII in `.headers`. (I presume there's some reason why we need unicode strings here?)

Comment (Ms2ger) [reply tools/wptserve/wptserve/handlers.py]: Because `template()` works on unicode strings, though maybe that was the wrong call.

Comment (Ms2ger) [reply tools/wptserve/wptserve/handlers.py]: No, I misread; `template()` works on bytes. `ResponseHeaders` handles both, but its callers expect text.

**Comment by gsnedders** [comment]
@Ms2ger can you rebase this onto master?

**Review by gsnedders — APPROVED** [review]

**Pull request merged**